src/Mod/Arch/ArchStairs.py

```diff
@@ -51,6 +51,10 @@ def QT_TRANSLATE_NOOP(ctxt,txt):
 #  This module provides tools to build Stairs objects.
 
 zeroMM = FreeCAD.Units.Quantity('0mm')
+<<<<<<< HEAD
```

Comment:
I've not seen this before. File is full of 
```
<<<<<<< HEAD
=======

>>>>>>> etc..etc..
```

Reply:
Yes, I found that upon PR - I think there are some conflicts which I simply do not understand and continue result in strange file?

I am googling maybe how to undo this and dowload again the latest ArchStairs.py since my last PR few days ago (last commits about 3 weeks ago though...).